inst/startup/merger.R

```diff
@@ -23,22 +23,29 @@ for(package in azbatchenv$packages){
 }
 parent.env(azbatchenv$exportenv) <- globalenv()
 
-results <- vector("list", numOfTasks)
-count <- 1
-tryCatch({
-  for(i in 1:N){
-    task_result <- paste0(AZ_BATCH_TASK_WORKING_DIR, "/result/", AZ_BATCH_JOB_ID, "-task", i, "-result.rds")
-    task <- readRDS(task_result)
-
-    for(t in 1 : length(task)){
-      results[count] <- task[t]
-      count <- count + 1
+enableMerge <- azbatchenv$enableMerge
+mergeFunc <- azbatchenv$mergeFunc
+
+if(typeof(mergeFunc) == "list" && enableMerge){
```

Comment:
does this mean I can't use rbind? e.g. summary <- foreach(i = 1:1000, .combine = rbind, .options.azure = opt)